Spec/Auth.swift

```diff
@@ -1748,21 +1747,18 @@ class Auth : QuickSpec {
                     }
 
                     waitUntil(timeout: testTimeout) { done in
-                        delay(tokenParams.ttl + 1.0) {
```

Comment:
This test also should work as it was before, or else we're breaking [RSA10g](http://docs.ably.io/client-lib-development-guide/features/#RSA10g).